### DISCUSSION TITLE
Questions regarding enforcement

### DISCUSSION THREAD

**Discussion by wmoussa-gc** [cds-snc/docs#6] [General]
From @CalvinRodo 

1. Who is responsible for making sure employees follow these RFC's?
2. What happens if folks don't follow the RFC?

**Comment by dj2** [comment]
I can see some of these coming up in response to need. I'm not sure how prescriptive we need to be to start.

1. We as a community are responsible. During code reviews, design reviews, etc. If there are practices we think are good as a community that aren't being followed then there should at least be a discussion as to why not.

2. If an RFC isn't being followed by the majority of folks, then I'd argue the RFC is wrong, needs to be retired and possibly a followup RFC written that captures how the community is working. I don't think an RFC should ever be considered static, these can, and probably should change as circumstances change.
